openlr_dereferencer/maps/a_star/tools.py

```diff
@@ -18,36 +18,9 @@ def heuristic(current: Node, target: Node) -> float:
     return distance(current.coordinates, target.coordinates)
 
 
-def reconstruct_path(reader: MapReader, came_from: Dict[Any, int], current: Node) -> List[Line]:
-    "When a path is found, it has to be reconstructed from the way the A* algorithm took."
-    cur = current
-    total_path = []
-
-    while cur.node_id in came_from.keys():
-        line = reader.get_line(came_from[cur.node_id])
-        total_path.insert(0, line)
-        cur = line.start_node
-    return total_path
-
-
 T = TypeVar("T")
```

Comment:
+1 for simplifying the module interface and removing `find_minimum`!

I think this TypeVar can vanish then as well.